change/@internal-react-components-9a822f88-c39d-4d91-9b15-36af563aa5b7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "comment": "add tooltipVideoLoadingContent to Camera strings",
+  "type": "none",
```

Comment:
@JamesBurnside this is defined by default but what should we set it to now?
Same question for "dependentChangeType"

Reply:
I haven't had time to update the options to minor/patch/none yet so easiest to leave as prerelease  - but I'll be going through the existing ones anyway to update them to patch or minor so wouldn't worry too much about it